bestie/server/testresult.go

```diff
@@ -142,77 +141,89 @@ func handleTestResultEvent(bazelBuildEvent bes.BuildEvent, streamId *build.Strea
 		// Attempt to read the zip file failed.
 		return fmt.Errorf("Error reading %s file: %w", outFileName, readErr)
 	}
-	if len(fileBytes) == 0 {
-		// Something went wrong reading zip file -- no data.
-		return fmt.Errorf("Error reading %s file: no data", outFileName)
-	}
 	// Process test metrics output file(s).
 	// Each output file contains a single (potentially large) protobuf message.
 	// For now, it's up to the client to split large metric datasets into multiple .metrics.pb files.
-	if err := extractZippedFiles(stream, fileBytes); err != nil {
+	if err := extractZippedFiles(stream, zipFileCloser); err != nil {
```

Comment:
nit: maybe extractZippedFiles gets renamed to `processZip`, since it doesn't extract all files anymore?

Reply:
OK